### PR TITLE
Homework2: Vlad Sheydenkov

### PR DESCRIPTION
**Pull request by vladsheydenkov**
### PR checklist
Please make sure you've passed PR checklist
- [ ] my PR passes CI checks (flake8 and pycodestyle)
- [ ] my PR named according to guidelines.
- [ ] my PR contains complete tasks descriptions for all tasks
- [ ] my PR made from the fork feature branch to maxim homework# branch.
- [ ] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore

### CODE REVIEW

File: src/homework2/task3.py [hunk 1]
```diff
@@ -14,7 +14,12 @@ def sub_string(str_):
     """
 
     # write your code here
-    return ''  # write return value here
+    new_str_ = ""
+    marks = ['.', ',', '!', '?', '-', ':', '(', ')', '"', '\'', ';', ' ']
```
Comment (belovmd) [inline src/homework2/task3.py:18]: redundant checks. we shouldn't delete punctuation marks here

File: src/homework2/task5.py [hunk 2]
```diff
@@ -11,7 +11,12 @@ def fibonacci(n):
     """
 
     # write your code here
-    return ''  # write return value here
+    fibonachi_list = [1, 2]
```
Comment (belovmd) [inline src/homework2/task5.py]: looks too memory consuming

**Review by belovmd — COMMENTED** [review]
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-summer/blob/master/Homework%20review/Homework2.md):
- 11
- 12
- 18

File: src/homework2/task3.py (hunk 1 above)
Comment (vladsheydenkov) [reply src/homework2/task3.py:18]: fixed

**Review by belovmd — COMMENTED** [review]
missed advanced tasks 
7.5

**Comment by belovmd** [comment]
missed advanced

**Pull request closed without merging**